src/test/scala/io/iohk/ethereum/consensus/ConsensusSpec.scala

```diff
@@ -400,6 +400,109 @@ class ConsensusSpec extends AnyFlatSpec with Matchers with ScalaFutures {
     blockchainReader.getBestBlock().get shouldEqual newBlock3WithOmmer
   }
 
+  it should "dequeue blocks where there is an execution error" in new EphemBlockchain {
```

Comment:
Those test are not strictly necessary in that PR as it does not touch the blockQueue but they make sure that I did not change the way the blockQueue is handled here and in the next PR.

Reply:
👍